src/main/java/com/oltpbenchmark/benchmarks/noop/procedures/NoOp.java

```diff
@@ -38,7 +38,7 @@
 
     // The query only contains a semi-colon
     // That is enough for the DBMS to have to parse it and do something
-    public final SQLStmt noopStmt = new SQLStmt(";");
+    public final SQLStmt noopStmt = new SQLStmt("select 1");
```

Comment:
This is not the same as a NOOP since it requires evaluation. Is there a reason for this change?

Maybe we should rename it to the "DoNothing" benchmark and then have separate `SelectOne` and `SemiColon` procedures.

Reply:
NoOp was one of the missing test cases; unfortunately HSQLDB doesn't accept `;` as a valid sql statement.  my thinking was this was a reasonable fix but now that i'm looking at it i should have just done this with a hsqldb dialect file and not changed the java class.  is that a better approach?

Reply:
this has been resolved by reverting the change to NoOp.java and instead adding a dialect-hsqldb.xml file with a hsqldb friendly query